src/core/control/tools/Selection.cpp

```diff
@@ -17,29 +17,51 @@ Selection::~Selection() = default;
 
 //////////////////////////////////////////////////////////
 
-RectSelection::RectSelection(double x, double y) {
+RectSelection::RectSelection(double x, double y, bool multiLayer) {
```

Comment:
Please use initializer lists in the constructors. Moreover `multiLayer` should be passed to the constructor of `Selection`, I think.